astrospec/spectrum1d.py

```diff
@@ -8,10 +9,18 @@
 
 from warnings import warn
 
+from .wcs import WCSTable
+
 
 class Spectrum1D(NDSlicingMixin, NDIOMixin, NDData):
-    """A 1 dimensionan spectrum.
-    
+    """A 1-dimensional spectrum.
+
+    Attributes
+    ----------
+
+    Methoods
+    --------
+
```

Comment:
Then I suggest that we add those when we have them. It's not clear to me yet what attribute we would document here.
The WCS maybe? That requires a full chapter...